Pull-requests-at-Oppia.md

```diff
@@ -0,0 +1,64 @@
+
+Learn more about skipping a [Travis CI build](https://docs.travis-ci.com/user/customizing-the-build/#skipping-a-build) and skipping a [Circle CI build](https://circleci.com/docs/2.0/skip-build/#skipping-a-build).
+
+## Labeling pull requests
+
+While contributing to Oppia, you will need to add different labels to issues or pull requests which you are working on. However, not all labels are allowed on issues and pull requests. Below are labels which can be applied to pull requests:
+
+* `dependencies`: Should be added to pull requests that updates one or more dependencies.
+* `PR: Affects datastore layer`: Indicates that a PR changes the datastore layer. Adding this label notifies the developers in charge of datastore stability so they can review the PR.
+* Changelog (labels containing `PR CHANGELOG`): Each PR should have one of these labels added to indicate what project this PR applies to. The developer mentioned in the label will be assigned to review the PR first.
+* Labels starting with `PR: don't merge`: Indicate that some problem with the PR needs to be addressed before merging. For example, one of these labels gets added if the PR author hasn't signed the CLA yet.
+* `PR: require post-merge sync to HEAD`: Should only be applied to pull requests which when merged will require all other open pull requests to be updated with the develop branch.
+* `PR: for current release`: Identifies PRs that need to be included in the release that is currently in progress. This label is used by the release team to identify PRs to cherry-pick into the release.
+* `PR: released`: Identifies PRs that have been cherry-picked into the release. Should only be used by the release team.
+* `REVIEWERS: Please add changelog label`: When a new contributor opens a PR, they don't have permission to add labels. In that case, Oppiabot adds this label to tell reviewers to assign the appropriate changelog label.
+* `stale`: This label gets added by oppiabot to PRs that have been inactive for a week. They will be automatically closed after 4 more days of inactivity.
+* `PR: LGTM`: Indicates that a PR has all necessary approvals. The PR can be merged as soon as the CI checks pass.
+* Other labels starting with `PR`.
+
+A complete list of labels can be found [here](https://github.com/oppia/oppia/labels).
+
+## Working on multiple pull requests
+
+After you've submitted a PR, and are waiting for it to get reviewed, you might want to start working on a new issue in the meantime. Git allows you to do this seamlessly! Always prioritize following up on your existing PRs and getting them ready for the next round of review. In particular, only start working on new issues/PRs after **all** the other PRs you're working on are in the "waiting for review" stage. This helps avoids creating a logjam in the build queues.
```

Comment:
The "Git allows you to do this seamlessly!" part seems a bit of a throwaway. Either elaborate on it or link it to a separate page -- it's a reasonable "new contributor confusion" that they aren't sure how to make a parallel PR and we should try to explain it somehow.

The next point is a separate point and should be given its own prominence (perhaps as a separate paragraph).

Reply:
done